middle_end/flambda2/numbers/targetint_31_63.ml

```diff
@@ -14,280 +14,149 @@
 (*                                                                        *)
 (**************************************************************************)
 
-let () =
-  match Targetint_32_64.num_bits with
-  | Sixty_four -> ()
-  | Thirty_two ->
-    if Flambda_features.flambda2_is_enabled ()
-    then Misc.fatal_error "Flambda 2 does not yet support 32-bit compilation"
+(* CR-someday mshinwell/gbury: maybe we might want to consider adding some more
+   checks in some of the conversions functions to be more safe and more
+   consistent in the handling of overflows ? For instance One_bit_fewer.of_int
+   silently truncates the input int to make it fit, whereas we probably want to
+   make it produce an error ? *)
 
-(* CR mshinwell/gbury: maybe we might want to consider adding some more checks
-   in some of the conversions functions to be more safe and more consistent in
-   the handling of overflows ? For instance One_bit_fewer.of_int silently
-   truncates the input int to make it fit, whereas we probably want to make it
-   produce an error ? *)
-
-module Imm = struct
-  module T0 = struct
-    include Numeric_types.Int64
-
-    let compare = Int64.compare
-
-    let equal = Int64.equal
-
-    let hash = hash
+module T0 = struct
+  include Numeric_types.Int64
 
-    let print = print
+  let compare = Int64.compare
```

Comment:
these definitions (including `hash` and `equal`) are not needed since `Numeric_types.Int64` already defines them

Reply:
Ah yes, well spotted - there were a lot of these.